src/app/portlets/dot-edit-page/layout/components/dot-theme-selector/dot-theme-selector.component.scss

```diff
@@ -26,6 +26,10 @@ $theme-item-dimension: 240px;
         font-size: $font-size-large;
         margin-top: $basic-padding * 5;
     }
+
+    .ui-dialog .ui-dialog-content {
+        padding: 17px $basic-padding * 4 $dialog-padding;
```

Comment:
If this is for fixing the space between the title and the content, we should move this to the dialog itself so all the dialogs have the same space.